src/services/db/CommitServiceGitFile.ts

```diff
@@ -0,0 +1,84 @@
+import UserWithSiteSessionData from "@root/classes/UserWithSiteSessionData"
+import isFileAsset from "@root/utils/commit-utils"
+import { isReduceBuildTimesWhitelistedRepo } from "@root/utils/growthbook-utils"
+
+import GitFileSystemService from "./GitFileSystemService"
+import { STAGING_BRANCH } from "./GitHubService"
+
+/**
+ * Responsibilities of this class
+ * 1. Creates all commits to staging
+ * 2. Creates non-asset related commits to staging-lite
+ */
+export default class CommitServiceGitFile {
```

Comment:
Should this `extends GitFileSystemService`?

Reply:
nope, there is no need to here + aim is to not use this as subsitude!